README.md

```diff
@@ -106,6 +106,13 @@ tagging = client.tags(tag_id).taggings.get(tagging_id)
 
 # Delete a Tagging
 client.tags(tag_id).taggings.delete(tagging_id)
+
+# look up events in an event campaign 'foo'
+event_campaign_id = 'foo'
```

Comment:
Should we use something that looks like an URN here, since that's the format (i think) ActionNetwork uses for identifiers?